.github/workflows/docker-jupyter-image.yml

```
@@ -0,0 +1,60 @@
+name: ghcr-jupyter
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  build-amd64:
+    runs-on: ubuntu-latest
+    if: github.repository == 'stfc/janus-core'
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: 'clone the repo'
+        uses: actions/checkout@v4
+      - name: 'login to ghcr'
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: ./containers
+          file: ./containers/Dockerfile.jupyter
+          push: true
+          tags: |
+            ghcr.io/stfc/janus-core/jupyter:amd64-${{ github.sha }}
+            ghcr.io/stfc/janus-core/jupyter:amd64-latest
+  build-arm64:
+    runs-on: ubuntu-latest-arm
```

```suggestion
    runs-on: ubuntu-24.04-arm
```
I don't think latest exists, unfortunately: https://github.com/orgs/community/discussions/148648#discussioncomment-11858287